module/VuFind/src/VuFind/Search/Solr/HideFacetValueListener.php

```diff
@@ -131,20 +132,29 @@ public function onSearchPost(EventInterface $event)
     protected function processHideFacetValue($event)
     {
         $result = $event->getParam('command')->getResult();
-        $facets = $result->getFacets()->getFieldFacets();
+        $facets = $result->getFacets();
```

Comment:
Is this listener no longer Solr-specific thanks to this standardization? Should we consider renaming it and wiring it up to other search backends? (Happy to do that as a follow-up PR if necessary).

Reply:
I think so, yes. That would make sense.

Reply:
I've put a TODO checkbox at the top to remind myself to do this as a follow-up after we merge this PR.